Training24/Program.c

```diff
@@ -1,15 +1,36 @@
-// ------------------------------------------------------------------------------------
+// --------------------------------------------------------------------------------
 // Training ~ A training program for new joiners at Metamation, Batch - July 2024.
 // Copyright (c) Metamation India.
-// -------------------------------------
-//  Dhaarani Mohanasundaram
-// -------------------------------------
-// Standard Program file in Main
+// Dhaarani Mohanasundaram
 // -------------------------------------
 // Program.c
-// Program on main branch.
-// ------------------------------------------------------------------------------------
-
-const char* GetMessage() {
-return "Hello, World!";
+// Program on A5 branch.
+// Program has functions for sorting an array and searching an element in array.
```

Comment:
Avoid mentioning 'program' and write only the functionality of the code file. Apply this wherever possible.
```suggestion
// Implements bubble sort and binary search
```